Makefile

```diff
@@ -298,3 +315,41 @@ clean: clean-release-manifest clean-bundle
 	@go clean -i $(PKGS)
 	@echo "Deleting image "$(OPERATOR_IMG)
 	@docker rmi -f $(OPERATOR_IMG) registry.access.redhat.com/ubi9-minimal:latest
+
+px-resource-gateway-proto-build-docker-proto:
+	docker build -t pure-artifactory.dev.purestorage.com/px-docker-dev-virtual/dgoel/px-resource-gateway-proto --network=host -f Dockerfile.proto .
+
+px-resource-gateway-docker-proto:
```

Comment:
Move this to a Makefile in the `proto` folder instead. Change it to a `proto:` target then have a `$(MAKE) -C proto` to execute it